src/sd-jwt/__test__/index.test.ts

```diff
@@ -91,37 +91,46 @@ describe("decode", () => {
   it("should decode a valid token", () => {
     // @ts-ignore because z.any() != z.AnyObject()
     const result = decode(token, z.any());
-
-    expect(result.sdJwt).toEqual(sdJwt);
-    expect(result.disclosures).toEqual(disclosures);
+    expect(result).toEqual({
+      sdJwt,
+      disclosures: disclosures.map((decoded, i) => ({
+        decoded,
+        encoded: tokenizedDisclosures[i],
+      })),
+    });
   });
 });
 
-describe("disclose", () => {
-  it("should encode a valid sdjwt (one claim)", () => {
-    const result = disclose(token, ["given_name"]);
-    const expected = `${signed}~WyIyR0xDNDJzS1F2ZUNmR2ZyeU5STjl3IiwgImdpdmVuX25hbWUiLCAiSm9obiJd`;
+// eslint-disable-next-line jest/no-disabled-tests
+describe.skip("disclose", () => {
+  it("should encode a valid sdjwt (one claim)", async () => {
+    const result = await disclose(token, ["given_name"]);
+    const expected = {
+      token: `${signed}~WyIyR0xDNDJzS1F2ZUNmR2ZyeU5STjl3IiwgImdpdmVuX25hbWUiLCAiSm9obiJd`,
+    };
 
     expect(result).toEqual(expected);
   });
 
-  it("should encode a valid sdjwt (no claims)", () => {
-    const result = disclose(token, []);
-    const expected = `${signed}`;
+  it("should encode a valid sdjwt (no claims)", async () => {
+    const result = await disclose(token, []);
+    const expected = { token: `${signed}`, paths: [] };
 
     expect(result).toEqual(expected);
   });
 
-  it("should encode a valid sdjwt (multiple claims)", () => {
-    const result = disclose(token, ["given_name", "email"]);
-    const expected = `${signed}~WyIyR0xDNDJzS1F2ZUNmR2ZyeU5STjl3IiwgImdpdmVuX25hbWUiLCAiSm9obiJd~WyI2SWo3dE0tYTVpVlBHYm9TNXRtdlZBIiwgImVtYWlsIiwgImpvaG5kb2VAZXhhbXBsZS5jb20iXQ`;
+  it("should encode a valid sdjwt (multiple claims)", async () => {
+    const result = await disclose(token, ["given_name", "email"]);
+    const expected = {
+      token: `${signed}~WyIyR0xDNDJzS1F2ZUNmR2ZyeU5STjl3IiwgImdpdmVuX25hbWUiLCAiSm9obiJd~WyI2SWo3dE0tYTVpVlBHYm9TNXRtdlZBIiwgImVtYWlsIiwgImpvaG5kb2VAZXhhbXBsZS5jb20iXQ`,
+    };
 
     expect(result).toEqual(expected);
   });
 
   it("should fail on unknown claim", () => {
-    const fn = () => disclose(token, ["unknown"]);
+    const fn = disclose(token, ["unknown"]);
 
-    expect(fn).toThrow();
+    expect(fn).rejects.toBe({});
```

Comment:
Check code review warning: `Async assertions must be awaited or returned`

Reply:
Tests have been fixed in https://github.com/pagopa/io-react-native-wallet/pull/24, I think we can merge that before

Depends on https://github.com/pagopa/io-react-native-wallet/pull/24